### DISCUSSION TITLE
"error": "accepting stream failed: timeout: no recent network activity"}

### DISCUSSION THREAD

**Discussion by ShahryarShabani** [apernet/hysteria#939] [Q&A] [answered]
Hello guys 

I have a problem: after some minutes I don't use the hysteriav2 VPN, the client gets disconnected.
how can I fix this problem that users always be connected?

my config file: 

listen: :443

tls:
  cert: /etc/hysteria/ca.crt
  key: /etc/hysteria/ca.key

quic:
  initStreamReceiveWindow: 8388608
  maxStreamReceiveWindow: 8388608
  initConnReceiveWindow: 20971520
  maxConnReceiveWindow: 20971520
  maxIdleTimeout: 120s
  maxIncomingStreams: 1024
  disablePathMTUDiscovery: false

  #bandwidth:
  #up: 150 mbps
  #down: 150 mbps
ignoreClientBandwidth: true
#obfs:
#  type: salamander
#  salamander:
#    password: v8Wt\1v_HT0TbTE1Cq9^42Fi

disableUDP: false
udpIdleTimeout: 60s

auth:
  type: userpass
  userpass:
    test: test

masquerade:
  type: string
  string:
    content: Request timeout
    headers:
      content-type: text/plain
      custom-stuff: Request timeout
    statusCode: 408

resolver:
  type: tls
  tcp:
    addr: 1.1.1.1:53
    timeout: 4s
  udp:
    addr: 8.8.8.8:53
    timeout: 4s
  tls:
    addr: 8.8.8.8:853
    timeout: 10s
    sni: cloudflare-dns.com
    insecure: true
  https:
    addr: 1.1.1.1:443
    timeout: 10s
    sni: cloudflare-dns.com
    insecure: false

outbounds:
  - name: warp
    type: socks5
    socks5:
      addr: 127.0.0.1:60000
acl:
  inline:
    - reject(all, udp/443)
    - reject(geosite:all)
    - reject(geoip:ir)
    - reject(geosite:ads)
    - reject(10.0.0.0/8)
    - reject(172.16.0.0/12)
    - reject(192.168.0.0/16)
    - reject(fc00::/7)
    - warp(35.184.0.0/13)
    - warp(suffix:spotify.com)
    - warp(suffix:google.com)
    - warp(suffix:chatgpt.com)
    - warp(157.240.0.0/17)
    - direct(all)
  # geoip: geoip.dat
  geosite: /etc/hysteria/geosite.dat
  # geoUpdateInterval: 168h

**Comment by ShahryarShabani** [comment, marked as answer]
I have found a solution and tested it. It is working great; I just need to add these configs. 

quic:
  initStreamReceiveWindow: 26843545
  maxStreamReceiveWindow: 26843545
  initConnReceiveWindow: 67108864
  maxConnReceiveWindow: 67108864
  maxIdleTimeout: 5s
  keepAlivePeriod: 5s
  maxIncomingStreams: 1024
  disablePathMTUDiscovery: false
  
  
  
  I hope this helps you guys

**Reply by summerKK** [reply]
you are my hero,bro

**Reply by qin9125** [reply]
> I have found a solution and tested it. It is working great; I just need to add these configs.
> 
> quic: initStreamReceiveWindow: 26843545 maxStreamReceiveWindow: 26843545 initConnReceiveWindow: 67108864 maxConnReceiveWindow: 67108864 maxIdleTimeout: 5s keepAlivePeriod: 5s maxIncomingStreams: 1024 disablePathMTUDiscovery: false
> 
> 太棒了，解决了我的 hy2 节点丢失问题

**Comment by makintosh777** [comment]
can you send json please
